packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js

```diff
@@ -0,0 +1,71 @@
+import { settings } from 'carbon-components';
+const { prefix } = settings;
+
+const _htmlTagRegex = /<.*?>/g;
+const _doubleSpaceRegex = /\s{2,}/g;
+const _italicRegex = /[_*](.*?)[_*]/g;
+const _boldRegex = /[_*]{2}(.*?)[_*]{2}/g;
+
+/**
+ * Removes any html tags from a string and keeps inner text if any
+ *
+ * @param {string} str String to be checked for html tags
+ * @returns {string} String with html tags stripped out
+ */
+const _removeHtmlTags = str => str.replace(_htmlTagRegex, '');
+
+/**
+ * Checks for double spaces between words in a string and replaces for single spaces
+ *
+ * @param {string} str String to be checked for double spaces
+ * @returns {string} String with html double spaces fixed
+ */
+const _fixDoubleSpaces = str => str.replace(_doubleSpaceRegex, ' ');
+
+/**
+ * Converts some markdown syntaxes into html
+ * It's not a full markdown-to-html converter
+ * It currently supports two syntaxes: Bold and Italic
+ * Bold: Double asterisk (**) or double underscore (__).
+ * Bold examples: **Lorem ipsum** __dolor__
+ * Italic: Single asterisk (*) or single underscore (_)
+ * Italic examples: _Lorem ipsum_ *dolor*
+ *
+ * @param {string} str String to convert to html
+ * @param {object} options Object with options for the conversion
+ * @param {boolean} options.italic Defines if should convert italic
+ * @param {boolean} options.bold Defines if should convert bold
+ * @param {boolean} options.useCarbonClasses If true uses carbon typography classes
```

Comment:
If these are optional you should update the JSDoc to show those. Also show the default values:

```
 * @param {boolean=} [options.italic=false] Defines if should convert italic
 * @param {boolean=} [options.bold=false] Defines if should convert bold
 * @param {boolean=} [options.useCarbonClasses=true] If true uses carbon typography classes
```

See: https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

Reply:
Right, also the options object itself is optional so it would look like the below right?

```
* @param {string} str String to convert to html
 * @param {object} [options={}] Object with options for the conversion
 * @param {boolean} [options.italic=false] Defines if should convert italic
 * @param {boolean} [options.bold=false] Defines if should convert bold
 * @param {boolean} [options.useCarbonClasses=true] If true uses carbon typography classes
```

Reply:
And I had the `useCarbonClasses ` be default as `true` as you put out in the example. I think it makes!

Reply:
@emerson-pereira for optional, you would need to add an `=` equal sign after the param type (e.g. `{boolean=}`)

Reply:
as talked on slack equal sign is not necessary in this syntax